src/omnicore/doc/release-notes.md

```diff
@@ -64,498 +49,132 @@ Omni Core is based on Bitcoin Core 0.10.4 and can be used as replacement for Bit
+Other API extensions
+--------------------
 
-Checkpoints and faster initial transaction scanning
----------------------------------------------------
+An optional parameter `height` can be provided, when using [omni_decodetransaction](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_decodetransaction), which is used to determine the parsing rules. If no `height` is provided, the chain height is used as default.
 
-When Omni Core is used the very first time, or when Omni Core hasn't been used for a longer period, then historical blocks are scanned for Omni transactions. With the introduction of "consensus hashes", a significant speed improvement during the initial stage was possible by skipping known blocks without Omni transactions, and by comparing the resulting state with checkpoints. If the state of the client doesn't match the reference state, the client shuts down to prevent providing inaccurate data.
+When retrieving feature activation transactions with [omni_gettransaction](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_gettransaction), then additional fields are included in the result: `"featureid"`, `"activationblock"` and `"minimumversion"`.
 
-Omni Core was intentionally not delivered with a snapshot of the most recent state, and transactions are fully verified in any case. One design goal of Omni Core has always been to minimize the role of central parties (in this case: the ones providing a snapshot or checkpoints), and to further minimize the dependency on a hardcoded list of information, the configuration option `-omniseedblockfilter=0` can be used to disable the skipping of blocks without Omni transactions.
+The Omni Core client version is now also exposed under the new key `"omnicoreversion"`, as well as inter via `"omnicoreversion_int"`, when using [omni_getinfo](https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/src/omnicore/doc/rpc-api.md#omni_getinfo). The old key `"mastercoreversion"` remains for compatibility in this version.
 
-The configuration option `-overrideforcedshutdown` can be used to prevent Omni Core from shutting down in case of a checkpoint mismatch. Using the latter is generally not recommended, as this exposes the user to data, which isn't considered as valid by other participants.
+The field `"positioninblock"` was added to RPCs retrieving or listing Omni transactions to provide visibility into the order of an Omni transaction within a block.
 
-Disable-wallet mode
--------------------
+Increased OP_RETURN payload size to 80 bytes
+--------------------------------------------
 
-Since this version Omni Core can be build and used without wallet support.
+The maximum size for OP_RETURN outputs used to be 80 bytes, then got changed to 40 bytes to be on the safe side, and then was increased once again.
 
```

Comment:
Did we ever release a 80 byte OP_RETURN initially with class C or did we activate at 40?  If so perhaps it's just necessary to say it's been increased?